app/src/pages/inside/projectSettingsPageContainer/content/notifications/notifications.scss

```diff
@@ -15,9 +15,59 @@
  */
 
 .notifications-container {
-  margin-top: 44px;
+  margin-top: 24px;
+  width: 100%;
+  max-width: initial;
+
+  & .contentItems>div {
```

Comment:
Try to avoid selectors by nested tags. They are sensitive to changes in HTML.
I suggest to use specific class name selectors.